modin/core/execution/ray/common/utils.py

```diff
@@ -113,7 +115,16 @@ def initialize_ray(
     if not ray.is_initialized() or override_is_cluster:
         cluster = override_is_cluster or IsRayCluster.get()
         redis_address = override_redis_address or RayRedisAddress.get()
-        redis_password = override_redis_password or RayRedisPassword.get()
+        redis_password = (
+            (
+                ray_constants.REDIS_DEFAULT_PASSWORD
+                if cluster
+                else RayRedisPassword.get()
+            )
+            if override_redis_address is None
```

Comment:
I don't understand why do we want to check `override_redis_address` here

Reply:
This is copy-paste effects :smile: Renamed to `override_redis_password`